pkg/k8s/k8s.go

```diff
@@ -133,6 +133,7 @@ var resourceNames = []resourceName{
 	{"srvauthz", "serverauthorization", "serverauthorizations"},
 	{"srv", "server", "servers"},
 	{"ap", "authorizationpolicy", "authorizationpolicies"},
+	{"hp", "httproute", "httproutes"},
```

Comment:
We have previously discussed adding a shortname for httproute and decided it was not a good idea.

Full context here: https://github.com/linkerd/linkerd2/pull/10030#discussion_r1054768605

Reply:
I have removed httproute from resource name for now, but would eventually have to add it under full name. Do we also plan to keep it in plural names ?

Reply:
I think we can just use "httproute" as both the short name and the full name.